reflectapi/src/codegen/typescript.rs

```diff
@@ -1102,6 +1102,7 @@ fn build_implemented_types() -> HashMap<String, String> {
 
     implemented_types.insert("std::vec::Vec".into(), "Array<T>".into());
     implemented_types.insert("std::collections::HashMap".into(), "Record<K, V>".into());
+    implemented_types.insert("std::collections::BTreeMap".into(), "Record<K, V>".into());
```

Comment:
this type is lacking fallback... adding BTreeMap support should not affect typescript codegen and any other codegen code in general.